scripts/api/data/licenses/licensePDDL-1.0.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "PDDL-1.0",
+  "uri": "http://opendatacommons.org/licenses/pddl/1.0/",
+  "shortDescription": "Open Data Commons Public Domain Dedication & License 1.0",
```

Comment:
In DataCite Fabrica this is called "ODC Public Domain Dedication & License 1.0". It comes from https://github.com/datacite/bracco/blob/df4ac2030774324adeaa72e2978c29dfb463ee3d/app/spdx.js#L3475C14-L3475C56

(The uri is fine, by the way. There's only one.)